### DISCUSSION TITLE
Starting from scratch

### DISCUSSION THREAD

**Discussion by joewxboy** [open-horizon/open-horizon.github.io#60] [Style Guide]
After discussing with @Rene-Ch1  ...

Let's start with "reasonable, broad guidelines".  Let's also keep in mind that while we may not initially tackle broad issues related to accessibility and I18N/L10N, we should not implement any guidelines that would need to be undone later.

**Comment by joewxboy** [comment]
* Copyright notice should span from start year through current year and be updated every January
  * Current year example: Copyright Foo, 2020
  * Second year example: Copyright Foo, 2020 - 2021
* Headlines and titles should only have the first word and proper nouns capitalized
* Copy should be written in the third-person, active voice
* Sequences and steps should be enumerated ... using numbered points for each
* When referring to people who use Open Horizon, do not use the words "customer" or "user".  Instead, use "adopter".

**Reply by Rene-Ch1** [reply]
@joewxboy The third bullet...s/b active voice. The others are correct. 

And what do you think about numbering this list to facilitate making cmts on specific points?

**Comment by Rene-Ch1** [comment]
Link to the corp IBM style document - https://ibmdocs-test.mybluemix.net/docs/en/ibm-style
